server/cmd/mmctl/commands/oauth.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mattermost/mattermost/server/public/model"
+
+	"github.com/mattermost/mattermost/server/v8/cmd/mmctl/client"
+	"github.com/mattermost/mattermost/server/v8/cmd/mmctl/printer"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var OAuthCmd = &cobra.Command{
+	Use:   "oauth",
+	Short: "Management of OAuth2 apps",
+}
+
+var ListOAuthAppsCmd = &cobra.Command{
+	Use:     "list",
+	Short:   "List OAuth2 apps",
+	Long:    "list all OAuth2 apps",
+	Example: "  oauth list",
+	RunE:    withClient(listOAuthAppsCmdF),
+}
+
+func listOAuthAppsCmdF(c client.Client, command *cobra.Command, args []string) error {
+	// Fetch all apps with a very large limit so we get them all.
+	apps, _, err := c.GetOAuthApps(context.Background(), 0, 100000000)
```

Comment:
We can also add page, perPage flags here, and pass default values so that usage would be same but there will be a flexibility. WDYT?

Reply:
Makes senses to me 👍 I'll add paging